code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -43,6 +43,9 @@
 	AddComponent(/datum/component/squeak, list('sound/creatures/mousesqueak.ogg' = 1), 100, extrarange = SHORT_RANGE_SOUND_EXTRARANGE) //as quiet as a mouse or whatever
 
 /mob/living/simple_animal/mouse/handle_automated_action()
+#ifdef UNIT_TESTS // DO NOT EAT MY CABLES DURING UNIT TESTS
+	return
+#else
 	if(prob(chew_probability) && isturf(loc))
 		var/turf/simulated/floor/F = get_turf(src)
```

Comment:
else isnt needed as the previous statement returns. so you can end the if under the return. makes it cleaner

Reply:
Expected this to give an `unreachable statement` warning, fixed!